usaspending_api/api_docs/api_documentation/advanced_award_search/spending_by_award.md

```diff
@@ -0,0 +1,180 @@
+## [Spending by Award](#spending-by-award)
+**Route:** `/api/v2/search/spending_by_award/`
+
+**Method:** `POST`
+
+This route takes award filters and fields, and returns the fields of the filtered awards.
+
+### Request
+field: Defines what award variables are returned.
+
+filter: Defines how the awards are filtered.  The filter object is defined here.
+
+https://github.com/fedspendingtransparency/usaspending-api/wiki/Search-Filters-v2-Documentation
+
+limit(**OPTIONAL**): how many results are returned
+
+page (**OPTIONAL**):  what page of results are returned
+
+sort (**OPTIONAL**): Optional parameter indicating what value results should be sorted by. Valid options are any of the fields in the JSON objects in the response. Defaults to the first `field` provided.
```

Comment:
Give an example to clarify that it is a string and a response object key name: `Award ID`

Reply:
done